pir/pir-internal/src/main/java/com/duckduckgo/pir/internal/component/PirActionsRunnerFactory.kt

```diff
@@ -29,13 +32,16 @@ class PirActionsRunnerFactory @Inject constructor(
     private val dispatcherProvider: DispatcherProvider,
     private val repository: PirRepository,
     private val jsMessageHelper: JsMessageHelper,
+    private val currentTimeProvider: CurrentTimeProvider,
+    private val pixelSender: PirPixelSender,
 ) {
     /**
      * Every instance of PirActionsRunner is created with its own instance of [PirMessagingInterface] and [RealBrokerActionProcessor]
      */
     fun getInstance(
```

Comment:
nit: I'd call this `createInstance`. `getInstace` generally means get a singleton while we're creating always a new one